src/api/treeTrackerApi.js

```diff
@@ -186,7 +186,7 @@ export default {
   },
   getCaptureById(id) {
     try {
-      const query = `${API_ROOT}/api/${getOrganization()}trees/${id}`;
+      const query = `${CAPTURE_MATCH_API}/captures/${id}`;
```

Comment:
It's a bit confusing to call this `CAPTURE_MATCH_API` here. It masks that we're just using the treetracker-api to query captures.